controllers/favorites.js

```diff
@@ -28,45 +28,31 @@ module.exports = (app) => {
   });
 
   // Send a POST request to the database to create the recipes collection
-  app.post('/favorites/', (req) => {
+  app.post('/favorites', (req) => {
+    console.log("***************** HERE")
     const favoriteId = req.body.recipeId;
     const userId = app.locals.user.id;
 
-    // find recipe, add userId to usersWhoFavorited
-    RecipeSchema.findById(favoriteId, (errFindingFave, favoriteInDB) => {
```

Comment:
are you sure you want to delete this whole function? if we don't update recipe, we won't be able to ask a recipe "how many users favorited you", we'll have to crawl through all the users and then sift through "which recipes they've favorited"